packages/runtime-dom/src/jsx.ts

```diff
@@ -248,8 +248,9 @@ export interface HTMLAttributes extends AriaAttributes, EventHandlers<Events> {
   contextmenu?: string
   dir?: string
   draggable?: Booleanish
-  hidden?: Booleanish
+  hidden?: Booleanish | 'hidden' | 'until-found'
```

Comment:
```suggestion
  hidden?: Booleanish | '' | 'hidden' | 'until-found'
```

According to the spec it should also accept an empty string.